src/content/app/tools/blast/components/blast-species-selector/BlastSpeciesSelector.test.tsx

```diff
@@ -27,7 +27,33 @@ import blastFormReducer, {
 
 import BlastSpeciesSelector from './BlastSpeciesSelector';
 
-import speciesList from './speciesList';
+const speciesList = [
+  {
+    assembly_name: 'GRCh38.p13',
+    common_name: 'Human',
+    genome_id: 'a7335667-93e7-11ec-a39d-005056b38ce3',
+    scientific_name: 'Homo sapiens'
+  },
+  {
+    assembly_name: 'IWGSC',
+    common_name: null,
+    genome_id: 'a73357ab-93e7-11ec-a39d-005056b38ce3',
+    scientific_name: 'Triticum aestivum'
+  },
+  {
+    assembly_name: 'GRCh37.p13',
+    common_name: 'Human',
+    genome_id: '3704ceb1-948d-11ec-a39d-005056b38ce3',
+    scientific_name: 'Homo sapiens'
+  }
+];
```

Comment:
We have a function for generating popular species objects: https://github.com/Ensembl/ensembl-client/blob/dev/tests/fixtures/popular-species.ts

Reply:
I tried to use this, but now I see the out-of-scope variable referencing issue you suspected yesterday. :-(